### PR TITLE
50 standardize language

### PR DESCRIPTION
**Pull request by gvegayon**
- [x] Changes `hospitalizations` to `hospital admissions`.
- [x] Changes `IHR` to `infection_hosp_rate`.

### CODE REVIEW

File: docs/build/html/searchindex.js
Comment (dylanhmorris) [inline docs/build/html/searchindex.js:1]: do we want to track this?

File: docs/build/html/_sources/msei_reference/model.rst.txt
Comment (dylanhmorris) [inline docs/build/html/_sources/msei_reference/model.rst.txt:1]: Why `.rst.txt` rather than `.rst`?

File: model/docs/example-with-datasets.md [hunk 1]
```diff
@@ -339,7 +339,7 @@ hosp_model.run(
     num_samples=2000,
     num_warmup=2000,
     n_timepoints=dat2.shape[0] - 1,
-    observed_hospitalizations=dat2,
+    observed_admissions=dat2,
```
Comment (dylanhmorris) [inline model/docs/example-with-datasets.md:342]: should probably rename `dat2` to something more specific, but that can be a separate PR

File: model/src/pyrenew/latent/hospitaladmissions.py [hunk 2]
```diff
@@ -13,23 +13,22 @@
...
-    Parameters
-    ----------
-    IHR : float, optional
-        The infected hospitalization rate. Defaults to None.
-    predicted : ArrayLike or None
-        The predicted number of hospital admissions. Defaults to None.
+        Parameters
+        ----------
+        infection_hosp_rate : float, optional
+            The infected hospitalization rate. Defaults to None.
```
Comment (dylanhmorris) [inline model/src/pyrenew/latent/hospitaladmissions.py:22]:
```suggestion
            The infection-to-hospitalization rate. Defaults to None.

File: model/src/pyrenew/latent/hospitaladmissions.py [hunk 3]
```diff
@@ -50,13 +49,23 @@ def __repr__(self):
 
 class InfectHospRate(RandomVariable):
     """
-    Infection to Hospitalization Rate
+        Infection to Hospitalization Rate
```
Comment (dylanhmorris) [inline model/src/pyrenew/latent/hospitaladmissions.py:52]:
```suggestion
        Infection-to-Hospitalization rate
```

File: model/src/pyrenew/latent/hospitaladmissions.py [hunk 4]
```diff
@@ -50,13 +49,23 @@ def __repr__(self):
...
+    <<<<<<< HEAD
```
Comment (dylanhmorris) [inline model/src/pyrenew/latent/hospitaladmissions.py:53]: Artifact from a git merge.
```suggestion
```

File: model/src/pyrenew/latent/hospitaladmissions.py [hunk 5]
```diff
@@ -50,13 +49,23 @@ def __repr__(self):
...
+
+        Methods
+        -------
+        validate(distr)
+            Validates distribution is Numpyro distribution
+        sample(**kwargs)
+            Produces a sample of the infection-hospitalization rate (IHR).
+    =======
+    >>>>>>> origin/main
```
Comment (dylanhmorris) [inline model/src/pyrenew/latent/hospitaladmissions.py:62]: Artifact from a git merge 
```suggestion
```

File: model/src/pyrenew/latent/hospitaladmissions.py [hunk 6]
```diff
@@ -98,7 +107,7 @@ def validate(distr: dist.Distribution) -> None:
 
     def sample(self, **kwargs) -> InfectHospRateSample:
         """
-        Produces a sample of the IHR
+        Produces a sample of the Infection-hospitalization Rate (IHR)
```
Comment (dylanhmorris) [inline model/src/pyrenew/latent/hospitaladmissions.py:110]:
```suggestion
        Produces a sample of an infection-to-hospitalization Rate (IHR)

File: model/src/pyrenew/model/rtinfectionsrenewalmodel.py [hunk 7]
```diff
@@ -207,13 +207,14 @@ def sample_infections_obs(
         name: str | None = None,
         **kwargs,
     ) -> tuple:
-        """Sample number of hospitalizations
+        """
+        Sample number of hospital admissions
```
Comment (dylanhmorris) [inline model/src/pyrenew/model/rtinfectionsrenewalmodel.py:211]: This function samples observed infections, not observed admissions. Docstring needs to be updated to reflect that.

```suggestion
        Sample observed infections according 
        to an observation process, if one has 
        been specified.
```

File: model/src/pyrenew/model/rtinfectionsrenewalmodel.py [hunk 8]
```diff
@@ -207,13 +207,14 @@ def sample_infections_obs(
...
 
         Parameters
         ----------
         predicted : ArrayLike
             The predicted infecteds.
-        observed_hospitalizations : ArrayLike | None, optional
+        observed_admissions : ArrayLike | None, optional
             The observed values of hospital admissions, if any, for inference. Defaults to None.
```
Comment (dylanhmorris) [inline model/src/pyrenew/model/rtinfectionsrenewalmodel.py:218]:
```suggestion
        observed_infections : ArrayLike | None, optional
            The observed infection values, if any, for inference. Defaults to None.
```

File: model/src/test/test_latent_admissions.py [hunk 9]
```diff
@@ -10,13 +10,13 @@
 from pyrenew.process import RtRandomWalkProcess
 
 
-def test_hospitalizations_sample():
+def test_admissions_sample():
     """
     Check that an InfectionObservation
```
Comment (dylanhmorris) [inline model/src/test/test_latent_admissions.py:15]:
```suggestion
    Check that a HospitalAdmissions latent process

File: model/src/test/test_latent_admissions.py [hunk 10]
```diff
@@ -10,13 +10,13 @@
...
     can be initialized and sampled from (deterministic)
```
Comment (dylanhmorris) [inline model/src/test/test_latent_admissions.py:16]:
```suggestion
    can be initialized and sampled from.
```

**Review by dylanhmorris — CHANGES_REQUESTED** [review]
Looks largely good. 

Main question is whether we really should be tracking `docs/build` in git.

Also some typo fixes.

File: docs/build/html/_sources/msei_reference/model.rst.txt
Comment (gvegayon) [reply docs/build/html/_sources/msei_reference/model.rst.txt:1]: We don't want to track `docs/build` at all. In #95 I take care of that.

**Review by dylanhmorris — APPROVED** [review]
